packages/lesswrong/server/resolvers/tagResolvers.ts

```diff
@@ -757,3 +760,79 @@ defineQuery({
     return { tags, totalCount };
   },
 });
+
+defineMutation({
+  name: "promoteLensToMain",
+  resultType: "Boolean",
+  argTypes: "(lensId: String!)",
+  fn: async (root, args: {lensId: string}, context) => {
+    const { lensId } = args;
+    if (!userIsAdminOrMod(context.currentUser)) {
+      throw new Error("Only admins can promote lenses to main");
+    }
+
+    // eslint-disable-next-line no-console
+    console.log(`Promoting lens ${lensId} to main`);
+    
+    // Find the lens and corresponding tag
+    const lensMultiDocument = await context.loaders.MultiDocuments.load(lensId);
+    if (!lensMultiDocument) throw new Error("Lens ID not found: "+lensId);
```

Comment:
We should also check that `lensMultiDocument` is in fact a lens (and not a summary).  Not sure how this would happen by accident, but if we ever ungate the functionality to non-admins, we won't want that possibility by default